bigfastapi/customer.py

```diff
@@ -0,0 +1,125 @@
+from datetime import datetime
+from typing import List
+from fastapi import APIRouter
+from bigfastapi.models import organisation_models
+from bigfastapi.utils.utils import generate_short_id
+from bigfastapi.schemas import customer_schemas
+from bigfastapi.models import customer_models
+import sqlalchemy.orm as orm
+import fastapi
+from bigfastapi.db.database import get_db
+from uuid import uuid4
+from fastapi.responses import JSONResponse
+from fastapi import status
+
+app = APIRouter(tags=["Customers 💁"],)
+
+
+
+
+@app.post("/customers", response_model=customer_schemas.CustomerCreateResponse)
+def create_customer(customer: customer_schemas.CustomerValidator, db: orm.Session = fastapi.Depends(get_db)):
+    organization = db.query(organisation_models.Organization).filter(organisation_models.Organization.name == customer.organization_name).first()
```

Comment:
You cannot use organisation name. ID, obviously

Reply:
fixed